internal/host/host.go

```diff
@@ -55,7 +55,7 @@ var InstallationProgressTimeout = map[models.HostStage]time.Duration{
 	models.HostStageStartWaitingForControlPlane: 60 * time.Minute,
 	models.HostStageInstalling:                  60 * time.Minute,
 	models.HostStageJoined:                      60 * time.Minute,
-	models.HostStageWritingImageToDisk:          60 * time.Minute,
+	models.HostStageWritingImageToDisk:          10 * time.Minute,
```

Comment:
why we should be so strict? What will we loose if we wait 30min? @avishayt @filanov @ronniel1 @oshercc ?

Reply:
that was @avishayt calculation.

Reply:
The ISO is 900MB.  Previously the timeout was set to 30 minutes (=30MB/minute) which in some extreme cases wasn't enough.  Now the limit is lowered to 15MB/minute.
This PR sets 10 minutes for every 5%, or 45MB.  That gives us 4.5MB/minute.  So we started from 30MB/min which was enough for all normal cases, and now we're allowing for 6X less bandwidth.  I think that's plenty.
Of course we can set it higher, but it's a balance between forgiving slow setups and failing quickly (if something bad happens, how long do you want to wait until you get some feedback?).